packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfApplication-CSharp/net6.0/App.xaml.cs

```diff
@@ -0,0 +1,20 @@
+﻿#if (!csharpFeature_ImplicitUsings)
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+#endif
+
+using System.Configuration;
+using System.Data;
+using System.Windows;
```

Comment:
Any reason why these aren't part of the implicit usings set? Even if those aren't in the SDK's global implicit using set, these can be scoped to the project (see https://docs.microsoft.com/dotnet/core/tutorials/top-level-templates#global-using-directives for more details).

One of the issues I can see here is that if a developer chooses not to use implicit usings, the usings will be unsorted, and this could be seen as annoynace by some developers (it'd certainly annoy me :)).

Reply:
As per the earlier discussions within the team : 

> There are two kinds of files that are typically created in a WPF application : 
> 1. Model files, for classes dealing with business logic, independent of UI - these tend to use
>            System.Collections.ObjectModel          // for ObservableCollection<T>  
>            System.ComponentModel                    // for INotifyPropertyChanged 
> 
> 2. View files, for classes that deal with UI, e.g. code-behind, they tend to use :
>            System.Windows                                  //   FrameworkElement , ResourceDictionary, RoutedEvent
>            System.Windows.Controls                   //    Button, DataGrid
>            System.Windows.Media                      //     Coor, Brush, Visual
>            System.Windows.Data                         //     Binding, IValueConverter, Validation
> 
> And although we can define different templates for these files, however, at the end when a project is compiled, there is a single global usings file, shared between all .cs files. So, it isn't possible to define different lists for different types of files.

That's why I did not modify the list as of now, and left it to what was enabled before.

Reply:
Makes sense. In this case it's worth considering doing this:
```cs
#if (!csharpFeature_ImplicitUsings)
using System;
using System.Collections.Generic;
using System.Configuration;
using System.Data;
using System.Linq;
using System.Threading.Tasks;
using System.Windows;
#else
using System.Configuration;
using System.Data;
using System.Windows;
#endif
```

Reply:
Made the changes.